### DISCUSSION TITLE
Ranking real-time calculation

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#24820] [Ideas] [answered]
Add a new feature--

Ranking can be like Quaver game in real-time calculation, avoiding boring rankings (most of the time, I see no movements in the rankings). This means that the score of each video in the ranking is calculated in real-time.

If this is very resource-intensive, consider only calculating the top 10 players.
If cannot download the videos from the server, then not calculating the ranking.
(MT)

**Comment by peppy** [comment, marked as answer]
I'm guessing you mean showing ghost scores in the gameplay leaderboard that count up as you play? If so, this is planned for local plays at very least.

**Reply by nana123126** [reply]
I think , it should be like this.
